_posts/2020-05-30-solo-socratic-2.md

```diff
@@ -0,0 +1,184 @@
+- [Multiprocess build support](https://github.com/bitcoin/bitcoin/pull/18677)
+
+### BIPs
+
+- [BIP 340 improvements](https://github.com/bitcoin/bips/pull/893)
+- [bip-322: simplify proposal to single proof case](https://github.com/bitcoin/bips/pull/903)
+- [bip-325: genesis block/message start](https://github.com/bitcoin/bips/pull/900)
+- [payjoin proposal](https://github.com/bitcoin/bips/pull/923)
+
+### eclair
+
+- [Eclair v0.4](https://github.com/ACINQ/eclair/releases)
+- [Use channel balance in path-finding](https://github.com/ACINQ/eclair/pull/1395)
+- [Compute max fee before route calculation](https://github.com/ACINQ/eclair/pull/1417)
+- [Use human readable features in configuration](https://github.com/ACINQ/eclair/pull/1385)
+
+### c-lightning
+
+- [Release v0.8.2.1: (Still) Scaling the Ethereum Blockchain](https://github.com/ElementsProject/lightning/releases/tag/v0.8.2.1)
+- [Release v0.8.2: Scaling the Ethereum Blockchain](https://github.com/ElementsProject/lightning/releases/tag/v0.8.2)
+- [Initial PSBT support](https://github.com/ElementsProject/lightning/pull/3738)
+- [Coin-movement notification type](https://github.com/ElementsProject/lightning/pull/3614)
+- [Watchtowers attempt 3: the full channelding](https://github.com/ElementsProject/lightning/pull/3659)
+- [Sort listinvoices and listsendpays by order of creation](https://github.com/ElementsProject/lightning/pull/3752)
+- [Offers preparation](https://github.com/ElementsProject/lightning/pull/3685)
+- [Pay: respect maxfeepercent when choosing a shadow route](https://github.com/ElementsProject/lightning/pull/3693)
+- [Plugin:added invoice creation event](https://github.com/ElementsProject/lightning/pull/3658)
+
+### lnd
+
+- [lnd v0.10.1-beta.rc3](https://github.com/lightningnetwork/lnd/releases/tag/v0.10.1-beta.rc3)
+- [lnd v0.10.0-beta](https://github.com/lightningnetwork/lnd/releases/tag/v0.10.0-beta)
+- [kvdb wrapper for etcd](https://github.com/lightningnetwork/lnd/pull/4015)
+- [add REST endpoints to all RPCs](https://github.com/lightningnetwork/lnd/pull/4251)
+- [walletrpc: add basic coin selection RPCs](https://github.com/lightningnetwork/lnd/pull/4303)
+- [multi: hold keysend payments](https://github.com/lightningnetwork/lnd/pull/4167)
+- [list expired invoices](https://github.com/lightningnetwork/lnd/pull/4168)
+- [dry run migration](https://github.com/lightningnetwork/lnd/pull/2668)
+- [add support for preset close address on inbound channels too](https://github.com/lightningnetwork/lnd/issues/4279)
+- [router+routerrpc: unify sendtoroute and payment responses](https://github.com/lightningnetwork/lnd/pull/4147)
+- [multi: Remove need for DerivePrivKey from watchtower, brontide, netann and sphinx package](https://github.com/lightningnetwork/lnd/pull/4227)
+- [lnrpc: add block height params to GetTransactions and add ListSweeps](https://github.com/lightningnetwork/lnd/pull/4139)
+
```

Comment:
```suggestion

### Wasabi

- [v1.1.11.1 release](https://github.com/zkSNACKs/WalletWasabi/releases/tag/v1.1.11.1)
- [PayJoin sending support](https://github.com/zkSNACKs/WalletWasabi/pull/3528)
- [PayJoin through clearnet as fallback](https://github.com/zkSNACKs/WalletWasabi/pull/3637)
- [Stricten mempool acceptance check to prevent too long mempool chains of CoinJoin](https://github.com/zkSNACKs/WalletWasabi/pull/3670)
- [Lower maximum CoinJoin coordinator fee now 0.3%](https://github.com/zkSNACKs/WalletWasabi/pull/3502)
- [HWI upgrade to fix trezor firmware upgrade issue](https://github.com/zkSNACKs/WalletWasabi/pull/3562)
```